src/logging.cpp

```diff
@@ -61,49 +109,13 @@ namespace logging {
 
     sink = boost::make_shared<text_sink>();
 
+#ifndef SUNSHINE_TESTS
```

Comment:
Is this really a good idea? I know that `inputtino` logs to `std::cout` (which is arguably not the best decision), but what if some other lib does too? Like some error before fataling or something.

Reply:
ugh, we are not capturing std::count, whoopsie...